src/prefs/KeyConfigPrefs.cpp

```diff
@@ -475,7 +475,9 @@ void KeyConfigPrefs::OnShow(wxShowEvent & event)
 {
    event.Skip();
 
-   if (event.IsShown())
+   // This is required to prevent a crash if Preferences 
+   // were opened without a project.
+   if (event.IsShown() && mView != nullptr)
```

Comment:
Maybe other uses of mView in this file should have null checks too

Reply:
I'm not aware of other crashes, but I will check.

This one I found out accidentally while testing